internal/handler/message/commands/commands.go

```diff
@@ -1,7 +1,15 @@
 package commands
 
+import "github.com/mymmrac/telego"
+
 const (
 	SendRules      = "rules"
 	SendUsefulInfo = "useful"
 	SendHelp       = "help"
 )
+
+var allCommands = []telego.BotCommand{
+	{Command: SendRules, Description: "правила спільноти"},
+	{Command: SendUsefulInfo, Description: "інформація про бота"},
+	{Command: SendHelp, Description: "корисна інформація (курси, документацію, платформи для навчання) по Go"},
```

Comment:
Краще трохи скоротити, бо на більшості клієнтів все одно не буде видно весь текст
```suggestion
	{Command: SendHelp, Description: "корисна інформація по Go"},
```